### PR TITLE
[Webapp] Bugfix: az webapp log deployment show - return deployment logs instead of log metadata

### LINKED ISSUE #14099: az webapp log deployment show should show actual logs instead of the log info
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62

**Describe the bug**
az webapp log deployment show should show actual logs instead of the log info

**To Reproduce**
az webapp log deployment show

**Expected behavior**
Show actual deployment logs instead of info about the deployment logs

Sample output:
[
  {
    "log_time": "2020...",
    "id": "id",
    "message": "log message",
    "type": 0,
    "details_url": null
  },
  {
    "log_time": "2020...",
    "id": "id",
    "message": "log message 2",
    "type": 0,
    "details_url": null
  }
]

### PR DESCRIPTION
**Pull request by calvinsID**
**Description**  
closes #14099 
- az webapp log deployment show should show actual logs instead of the log info
- Mark commands as in preview

**Testing Guide**  
`az webapp log deployment show`

Should have output of a list of logs similar to:

[
{
"log_time": "2020...",
"id": "id",
"message": "log message",
"type": 0,
"details_url": null
},
{
"log_time": "2020...",
"id": "id",
"message": "log message 2",
"type": 0,
"details_url": null
}
]

Rather than an object similar to

{
    "active": false,
    "author": "N/A",
    "author_email": "N/A",
    "complete": true,
    "deployer": "GITHUB_ZIP_DEPLOY",
    "end_time": "2020-06-25T19:04:18.8758562Z",
    "id": "id",
    "is_readonly": true,
    "is_temp": false,
    "last_success_end_time": "2020-06-25T19:04:18.8758562Z",
    "log_url": "https://calvin-githubactions-webapp.scm.azurewebsites.net/api/deployments/deploymentId/log",
    "message": "Created via a push deployment",
    "progress": "",
    "received_time": "2020-06-25T19:04:00.9811072Z",
    "site_name": "calvin-githubactions-webapp",
    "start_time": "2020-06-25T19:04:01.061354Z",
    "status": 4,
    "status_text": "",
    "url": "https://calvin-githubactions-webapp.scm.azurewebsites.net/api/deployments/deploymentId"
  }

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Webapp

File: src/azure-cli/azure/cli/command_modules/appservice/custom.py [hunk 1]
```diff
@@ -1974,34 +1974,39 @@ def show_diagnostic_settings(cmd, resource_group_name, name, slot=None):
...
+        deployment_log_url = '{}/api/deployments/{}/log'.format(scm_url, deployment_id)
+    else:
+        deployments_url = '{}/api/deployments/'.format(scm_url)
+        response = requests.get(deployments_url, headers=headers)
+
+        if response.status_code != 200:
+            raise CLIError("Failed to connect to '{}' with status code '{}' and reason '{}'".format(
+                deployments_url, response.status_code, response.reason))
+
+        sortedLogs = sorted(
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/appservice/custom.py:1995]: code style, sorted_logs

**Comment by qwordy** [comment]
ready to merge?

**Comment by calvinsID** [comment]
> ready to merge?

Ready, thanks!

**Pull request merged**